### PR TITLE
Vendor gardener/gardener@1.70.2

### PR DESCRIPTION
**Pull request by acumino**
**How to categorize this PR?**

/area control-plane
/kind enhancement
/platform equinix-metal

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Part of https://github.com/gardener/gardener/issues/7635

**Special notes for your reviewer**:

**Release note**:

```other operator
The following dependency has been updated:
- github.com/gardener/gardener 1.66.1 -> 1.70.2
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -8,8 +8,8 @@ require (
 	github.com/ahmetb/gen-crd-api-reference-docs v0.3.0
 	github.com/coreos/go-systemd/v22 v22.3.2
 	github.com/gardener/etcd-druid v0.15.3
-	github.com/gardener/gardener v1.66.1
-	github.com/gardener/machine-controller-manager v0.45.0
+	github.com/gardener/gardener v1.70.2
+	github.com/gardener/machine-controller-manager v0.48.1
```
Comment (schrodit) [inline go.mod:12]: is it also required to update the equinix machine prover: https://github.com/gardener/machine-controller-manager-provider-equinix-metal ?

**Review by schrodit — COMMENTED** [review]
@acumino is it also required to run gardener 1.70 as minimum supported version?

change is lgtm

File: go.mod (hunk 1 above)
Comment (acumino) [reply go.mod:12]: > is it also required to run gardener 1.70 as minimum supported version?

No.

> is it also required to update the equinix machine prover

Not with this version of Gardener. But for future versions of gardener ref- https://github.com/gardener/gardener/pull/7599#issuecomment-1459398514, MCM-provider will be required  to vendor new version for MCM.

**Review by schrodit — APPROVED** [review]
/lgtm

**Pull request merged**